src/styles/blocks/card.scss

```diff
@@ -0,0 +1,71 @@
+.card {
```

Comment:
It is a good practice to use flexbox for the card layout, as it is not mentioned in the code. Add 'display: flex; flex-direction: column;' to the .card class.